### DISCUSSION TITLE
Custom theme

### DISCUSSION THREAD

**Discussion by thomaskvael** [nativewind/nativewind#46] [Q&A] [answered]
Hi

I can only use the predefined tailwind styles.

If I try to create a custom color i.e. darkred it won't work.

```
theme: {
  extend: {
    colors: {
      darkred: '#b91c1c',
    },
  },
},
```

I also noticed that you can't use `className="red"`, `className="blue"` etc.

Is this expected behavior or can something be wrong with my setup?
I followed the basic babel setup with Expo.

**Comment by marklawlor** [comment, marked as answer]
Hi @thomaskvael, theme extension should be working, but it appears that you are incorrectly using Tailwind CSS. 

The [Text Color docs](https://tailwindcss.com/docs/text-color) show that the usage is `text-{color}-{variant}`. So `text-red-500` or `text-blue-500` will work.

Since you have extended your theme with a color with no variants, you can use it by simply removing the variant section. eg `className="text-darkred"`.

Because we use the official tailwind compiler, a good way to test this library is running the Tailwind CLI on your code base and inspecting the CSS output. If it compiles correctly then this library should work :+1:

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
